js/languages/en-US.json

```diff
@@ -806,6 +806,7 @@
   },
```

Comment:
Yeah no worries. I haven't tested thoroughly yet, but so far looking pretty good! I'll test more closely once we feel checkout is near feature complete.